app/lib/bk/compat/parsers/bitrise/workflows.rb

```diff
@@ -7,26 +7,29 @@ class Bitrise
       private
 
       def parse_workflow(wf_name, wf_config)
-        # Extend for other types of steps - approvals/waits for example.
-        BK::Compat::CommandStep.new(label: wf_name, key: wf_name).tap do |cmd_step|
-          wf_config['steps'].each do |step|
-            step_key, step_config = step.first
-            step_inputs = step_config['inputs'].reduce({}, :merge)
-            cmd_step << load_workflow_step(step_key, step_inputs)
-          end
+        stack_value = wf_config.dig('meta', 'bitrise.io', 'stack')
+        agents = stack_value ? { stack: stack_value } : {}
+        cmd_step = BK::Compat::CommandStep.new(label: wf_name, key: wf_name, agents: agents)
+        wf_config['steps'].each do |step|
+          step_key, step_config = step.first
+          cmd_step << translate_step(step_key, step_config)
         end
+
+        cmd_step
```

Comment:
why change what was already there?

```suggestion
        BK::Compat::CommandStep.new(label: wf_name, key: wf_name, agents: agents).tap do |cmd_step|
          wf_config['steps'].each do |step|
            step_key, step_config = step.first
            cmd_step << translate_step(step_key, step_config)
          end
        end
```